charts/matomo/templates/deployment.yaml

```diff
@@ -1,19 +1,21 @@
-apiVersion: apps.openshift.io/v1
-kind: DeploymentConfig
+apiVersion: apps/v1
+kind: Deployment
 metadata:
+  labels:
+    app: {{ .Values.mariadb.name }}
   annotations:
     template.alpha.openshift.io/wait-for-ready: "true"
-  name: mariadb
+  name: {{ .Values.mariadb.name }}
 spec:
-  replicas: 1
   selector:
-    name: mariadb
+    matchLabels:
+      app: {{ .Values.mariadb.name }}
   strategy:
-    type: Recreate
+    type: RollingUpdate
```

Comment:
This does not work in Rahti 1 with cinder volumes. Because the old pod is not killed till the new one is running. As cinder can only be mounted once, the deployment never ends.

Reply:
Changed to `Recreate` because of cinder volumes